readme.md

```diff
@@ -118,6 +118,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ColdFusion](https://github.com/seancoyne/awesome-coldfusion)
 - [Fortran](https://github.com/rabbiabram/awesome-fortran)
 - [.NET](https://github.com/quozd/awesome-dotnet)
+	- [.NET Core](https://github.com/thangchung/awesome-dotnet-core)
```

Comment:
I would set this to `- [Core](https://github.com/thangchung/awesome-dotnet-core)`. It's under .NET item so it's implied that it's about .NET.